DESCRIPTION

```diff
@@ -35,7 +35,9 @@ Suggests:
     rmarkdown,
     testthat (>= 3.0.3),
     vdiffr (>= 1.0.0),
-    withr
+    withr,
+    showtext,
```

Comment:
Suggest are not supposed to propagate to depending package but well... I guess this is a special case.

Reply:
> Suggest are not supposed to propagate to depending package but well... I guess this is a special case.

I was desperate... but it did actually help. Could you please add it to suggest of esqlabsR?